app/controllers/api/services_controller.rb

```diff
@@ -40,15 +40,15 @@ def usage_rules
   end
 
   def create
-    @service = collection.new # this is done in 2 steps so that the account_id is in place as preffix_key relies on it
+    @service = ServicePresenter.new(collection.new)
     creator = ServiceCreator.new(service: @service)
 
     if can_create? && creator.call(create_params)
       flash[:notice] = t('flash.services.create.notice', resource_type: product_or_service_type)
       onboarding.bubble_update('api')
       redirect_to admin_service_path(@service)
     else
-      flash.now[:error] = t('flash.services.create.error', resource_type: product_or_service_type)
+      flash.now[:error] = flash_message(@service.errors.keys.first, {scope: %i[flash services create errors]})
```

Comment:
:scream: What happened @didierofrivia ?

Reply:
This is the old version

Reply:
![image](https://user-images.githubusercontent.com/11318903/69251129-3a182800-0bb1-11ea-8b37-b2ed3fbaed97.png)

![image](https://user-images.githubusercontent.com/11318903/69251103-2a98df00-0bb1-11ea-9daa-98ffffdd99fa.png)

Reply:
oops!